### PR TITLE
Making sure models are rebuilt when fremen server is restarted

### PR DESCRIPTION
**Pull request by Jailander**
Needs this to work as expected: https://github.com/strands-project/fremen/pull/26

Should solve negative numbers issues

### CODE REVIEW

**Comment by Jailander** [comment]
Now it really needs this strands-project/fremen#26 to work

File: door_pass/scripts/door_prediction.py [hunk 1]
```diff
@@ -80,43 +89,51 @@ def MapCallback(self, msg) :
...
+    
+            rospy.loginfo('Building model for epoch range: %s' % self.range)
+    
+            start_time = time.time()  
+            #self.get_list_of_edges()
+            #elapsed_time = time.time() - start_time
+            #self._feedback.result = "%d edges found in %.3f seconds \nGathering stats ..." %(len(self.eids),elapsed_time)
+            #self._as.publish_feedback(self._feedback)
+            #self.gather_stats()        
+            self.create_models()
```
Comment (cdondrup) [inline door_pass/scripts/door_prediction.py:124]: So it creates models in the `fremen_restart_cb` and here? Seems a bit redundant.

**Comment by Jailander** [comment]
That is the action server callback, this is only called when the action server is called, the `fremen_restart_cb` is only called when for some reason the **fremenserver node** is started (the first time or because it has respawned). 
They are different functions because I don't want to set the actionserver status or publish feedback when it hasn't been called, or what other type of redundancy do you mean?

**Comment by Jailander** [comment]
I think @bfalacerda is testing on Betty we can wait for him to say if it works there before merging.

**Comment by bfalacerda** [comment]
oh, i missed this one sorry. I have the door prediction in the same launch file as fremen now though, if you tested the rebuilding in linda, I'll just add this and check if nothing breaks. Will let you know later

**Comment by Jailander** [comment]
Ok but don't forget  strands-project/fremen#26 you need this one if not it won't work

**Comment by hawesie** [comment]
@bfalacerda let me know if this is ready to merge

**Comment by bfalacerda** [comment]
It's fine , I haven't tested it restarting fremen but it sure doesn't break anything

**Comment by bfalacerda** [comment]
Cool, That's pretty significant savings according to my PhD in electrical And computing engineering

**Comment by bfalacerda** [comment]
Ah wrong place for this joke!

**Pull request merged**